db/mysql/schema.sql

```diff
@@ -286,6 +300,16 @@ CREATE TABLE `sch_usr_notification_email`
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 CREATE INDEX `ix_SCH_USR_NOTIFICATION_EMAIL_email_id` ON `sch_usr_notification_email` (`email_id`);
 
+
+CREATE TABLE `organization`
```

Comment:
1.organization comes only with authenticated user (in our case from sso)
2. there is no connection in the db between user and organization (user can
change his organization between authenticated sessions)
3. if the authenticated user does not have an organization, no organization
is saved for the entities you mentioned

On Fri, Mar 20, 2015 at 8:52 PM, Tomas Knap <notifications@github.com>
wrote:

> In db/mysql/schema.sql
> <https://github.com/UnifiedViews/Core/pull/282#discussion_r26870794>:
>
> > @@ -286,6 +300,16 @@ CREATE TABLE `sch_usr_notification_email`
> >  ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
> >  CREATE INDEX `ix_SCH_USR_NOTIFICATION_EMAIL_email_id` ON `sch_usr_notification_email` (`email_id`);
> >
> > +
> > +CREATE TABLE `organization`
>
> Table organization is created, but:
> 1) it is not prepopulated with any value? If not, where is the value for
> the organization taken?
> 2) Is this table used somehow? There are couple of tables (e.g. ppl_model)
> including column organization_id, but it seems to me that it is always
> null. For example I created new pipeline and the organization_id for the
> new pipeline was null.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UnifiedViews/Core/pull/282/files#r26870794>.
>